### PR TITLE
[drape] Fixed pattern's rounding.

### PR DESCRIPTION
**Pull request by vng**

### CODE REVIEW

File: drape/stipple_pen_resource.hpp [hunk 1]
```diff
@@ -24,7 +22,9 @@ using PenPatternT = buffer_vector<uint16_t, 2>;
 
 inline uint16_t PatternFloat2Pixel(double d)
 {
-  return static_cast<uint16_t>(std::round(d));
+  auto const px = static_cast<uint16_t>(std::round(d));
```
Comment (biodranik) [inline drape/stipple_pen_resource.hpp:25]: Isn't the result of `std::round` equal to the result of `static_cast<uint16_t>`?

**Review by biodranik — APPROVED** [review]
Did you test it?

File: drape/stipple_pen_resource.hpp (hunk 1 above)
Comment (vng) [reply drape/stipple_pen_resource.hpp:25]: No

**Comment by vng** [comment]
Yes, works fine when set m_ratio = 1 always.

**Comment by pastk** [comment]
Does it produce any visual change?

I don't really understand what cases does it affect..

**Comment by vng** [comment]
Before the fix (0.9, 0.4) was loaded as (1, 0) here
https://github.com/organicmaps/organicmaps/blob/master/drape/texture_manager.cpp#L444

**Comment by pastk** [comment]
Ok, there is no visual change (I've checked on my mdpi screen).
Looks like before the fix (1,0) were loaded from patterns.txt, but then at the runtime (1,1) was requested and texture uploaded separately (?).

**Pull request merged**